spec/models/validations/sales/setup_validations_spec.rb

```diff
@@ -113,11 +112,11 @@
           allow(Time).to receive(:now).and_return(Time.zone.local(2025, 1, 8))
         end
 
-        it "cannot create new logs for the previous collection year" do
+        it "cannot create new logs for the archived collection year" do
           record.update!(saledate: nil)
-          record.saledate = Time.zone.local(2024, 1, 1)
+          record.saledate = Time.zone.local(2023, 1, 1)
           setup_validator.validate_saledate_collection_year(record)
-          expect(record.errors["saledate"]).to include(match "Enter a date within the 24/25 collection year, which is between 1st April 2024 and 31st March 2025")
+          expect(record.errors["saledate"]).to include(match "Enter a date within the 23/24 or 24/25 collection years, which is between 1st April 2023 and 31st March 2025")
         end
```

Comment:
I imagine these tests might run a bit quicker if you just build a log with saledate: x on line 1 and feed that straight into the validator? Is there some important state in record ebfore the test starts?